src/Config.cpp

```diff
@@ -321,6 +317,11 @@ std::vector<ConfigItem<int>> CFG::getIntItems()
         {"websocket.loglevel", websocket.loglevel, 4096, [](const int &v) { return v > 0 && v <= 4096; }},
         {"websocket.port", websocket.port, 8089, validateInt65535},
         {"websocket.first_image_delay", websocket.first_image_delay, 100, validateInt65535},
+        // Next section contains old options, only necessary for your migration.
```

Comment:
Considering that a camera firmware upgrade wipes the previous config, I'm not so sure we need to worry about migrating anything?

Reply:
Please remove the migration logic. It's not needed